docs/src/md/kotlin.core/inheritance.md

```diff
@@ -80,6 +80,85 @@ As Kotlin is a language with single inheritance (only one supertype can be a cla
 
 TODO(Examples)
 
+### Inheritance delegation
```

Comment:
Remark: here the "Inheritance delegation" section is not located in the "Inheriting" section, but rather as an independent section directly inside the "Inheritance" chapter.

Apparently, if we are still considering places within the "Inheritance" chapter, it may seem meaningful to move the section to the "Classifier type inheritance", as, well, it does speak about the classifiers, but as I see it, "Classifier type inheritance" is more general, talking about *what* can actually be affected by inheritance, rather than about specifics of *how*.
The next section, "Matching and subsumption of declaration" describes which declarations can be considered as "matching", and thus a section about one specific case of inheritance ("Inheritance delegation") cannot be placed here or before it.
Next comes the "Inheriting" section, which, to be honest, just expands on the previous section, again, talking about general rules for inheritance.
The only section left is "Overriding", which is the only section finally introducing overriding and showing concrete examples. 

Perhaps it may be meaningful to put it after the "Overriding" section, especially since "Inheritance delegation" references it, and, as it seems, is even more specific.

Reply:
I think I would prefer to place it right after the "Classifier type inheritance", tbh.